sklearn/compose/_column_transformer.py

```diff
@@ -419,6 +419,9 @@ def fit_transform(self, X, y=None):
             sparse matrices.
 
         """
+        if isinstance(X, list):
+            X = np.array(X)
```

Comment:
I think using `check_array` from `utils/validation` would be safer/cleaner.

Reply:
Thanks for the tip! Added.